Dockerfile

```diff
@@ -170,6 +171,20 @@ RUN  --mount=target=/root/.cache,type=cache \
   CGO_ENABLED=0 go build -o /rootlesskit ./cmd/rootlesskit && \
   file /rootlesskit | grep "statically linked"
 
+# Based on https://github.com/containers/fuse-overlayfs/blob/v0.7.6/Dockerfile.static.ubuntu .
+# We use Ubuntu here because Alpine does not provide an apk package for libfuse3.a .
```

Comment:
Not super important but if it works on buster we could use that as we already use that image here.

Reply:
updated